docs/changes.rst

```diff
@@ -133,11 +135,13 @@ Released: not yet
 **Enhancements:**
 
 * Extend pywbem MOF compiler to search for dependent classes including:
+
   a) reference classes (classes defined in reference properties or parameters)
-  b)EmbeddedInstance qualifier classes if they are not compiled before the
-  classes that reference them is compiled. Previously the lack of these
-  dependent classes was ignored.  The compiler already searches for superclasses
-  if they are not compiled before their subclasses.
+
+  b) EmbeddedInstance qualifier classes if they are not compiled before the
+     classes that reference them is compiled. Previously the lack of these
```

Comment:
change "them is compiled" to "them are compiled"

Reply:
DONE